pg_backup_api/pg_backup_api/run.py

```diff
@@ -125,9 +129,13 @@ def main():
     args = p.parse_args()
     if hasattr(args, "func") is False:
         p.print_help()
+        ret = True
     else:
-        return args.func(args)
+        output, ret = args.func(args)
```

Comment:
```suggestion
        output, ret = args.func(args)
        if output:
            print(output)
```

I guess we should print the output :)